service/static/js-src/muk.institution.js

```diff
@@ -3571,6 +3573,13 @@ $.extend(edges, {
             }
         };
         this.synchronise = function() {
+            if(initialPageLoad) {
+                initialPageLoad = false
+                var institutionList = this.edge.result.buckets("institution");
+                for (var i = 0; i < 9; i++) {
```

Comment:
This loop it's the same for all the report pages. Can we agree on the default of 10 results? I'll change this in a second